harfbuzz_overflow_resolution.md

```diff
@@ -0,0 +1,157 @@
+# Introduction
+
+Several tables in the opentype format are formed internally by a graph of subtables. Parent node's
+reference their children through the use of positive offsets, which are typically 16 bits wide.
+For storage in the font file the graph must be given a topological ordering and then the subtables
+packed in serial according to that ordering. Since 16 bit offsets have a maximum value of 65,535 if
+the distance between a parent subtable and a child is more then 65,535 bytes then it's not possible
+for the offset to encode that edge.
+
+For many fonts with complex layouts it's not unusual for the GSUB/GPOS table to be larger than 65kb.
```

Comment:
you didn't mention GSUB/GPOS until now, they just popped out of nowhere. Perhaps introduce roughly what sorts of things layout *does* and reference the tables and link to the spec from paragraph 1 (or even add paragraph 0)?